### DISCUSSION TITLE
Remaining time not working anymore

### DISCUSSION THREAD

**Discussion by dalsmthme** [mriscoc/Ender3V2S1#610] [General]
I have updated to the latest firmware on my Ender 3 V2 Neo.  The remaining time field is no longer working.  It was working but on my last two prints it only displayed 0:00.  Could I be doing something wrong?

**Comment by mriscoc** [comment]
If your generated Gcode file includes M73 commands the internal timings are disabled to allows the more precise M73 plugin timing report. 
If printing via a host like Octoprint the time information must be provided via a plugin.

**Reply by dalsmthme** [reply]
Forgive me for I am new at the G-code side of things..... Where would I add this M73 Command?  I assume not in the start codes correct?  I was looking in post processing and I can turn on the Display Progress post script but that gives me the M117 command.

**Reply by dalsmthme** [reply]
Nevermind, I see now its a drop down where I can select M73.  I will give that a try.

**Reply by broderp1** [reply]
> Nevermind, I see now its a drop down where I can select M73. I will give that a try.

Where did you see this drop down and did it resolve the issue?

**Comment by broderp1** [comment]
I have this exact same issue and am greener than the author.   

You responded with:
"If your generated Gcode file includes M73 commands the internal timings are disabled to allows the more precise M73 plugin timing report."    

Does this refer to my Start Gcode in Cura?   And if I don't have any M73 commands in my Gcode, where do I go from here?  Further, if the M73 commands from the Gcode are more accurate, then does the firmware not take advantage of this?  Should I disable the code in Cura to allow the remaining time to work via the firmare calculations?

**Reply by broderp1** [reply]
UPDATE.   The issue self-corrected.   I don't know how accurate the remaining time is but it seems to be working on all prints.  I am noticing a few glitches here and there with this firmware, but overall, its a good update.

**Reply by mriscoc** [reply]
Thanks for the feedback. I've found that the remaining time is more accurate if you use a slicer plugin to send M73 R G-codes.

**Comment by mriscoc** [comment]
Check https://github.com/mriscoc/Ender3V2S1/wiki/Octoprint#display-print-progress
